FileUpload/src/Browse/FilePicker.jsx

```diff
@@ -4,8 +4,6 @@ const searchIcon = require("!raw!./img/search.svg");
 const fileIcon = require("!raw!./img/pages.svg");
 import { Scrollbars } from "react-custom-scrollbars";
 
-const notSpecifiedText = "<None Specified>";
```

Comment:
Out of curiosity, where is the default text now?

Reply:
Actually the real problem here is the "!raw!" prefix... I should investigate, since it won't work without webpack, I guess.